lib/plugins/filter/before_post_render/backtick_code_block.js

```diff
@@ -48,6 +49,14 @@ function backtickCodeBlock(data) {
           }
         }
       }
+
+      if (exclude_languages.includes(args)) {
+        // Only wrap with <pre><code class="lang"></code></pre>
+        options.wrap = false;
+        options.hljs = true;
```

Comment:
I haven't test this PR, my understanding is that, while enabling `hljs` also enables `wrap`, it is also possible to disable both at the same time.

Reply:
To be honest, I have read the code in [hexo-util](https://github.com/hexojs/hexo-util), but it seems that there have been some changes in `wrap` and `gutter` in  function `highlightUtil` [lib/highlight.js](https://github.com/hexojs/hexo-util/blob/master/lib/highlight.js):
```javascript
// newer version:

if (useHljs && !gutter) wrap = false;
if (gutter && !wrap) wrap = true; // arbitrate conflict ("gutter:true" takes priority over "wrap:false")

const before = useHljs ? `<pre><code class="${classNames}">` : '<pre>';
const after = useHljs ? '</code></pre>' : '</pre>';

if (!wrap) return `<pre><code class="${classNames}">${data.value}</code></pre>`;
```

```javascript
// older version
const before = useHljs ? `<pre><code class="hljs ${options.lang}">` : '<pre>';
const after = useHljs ? '</code></pre>' : '</pre>';

if (!wrap) return useHljs ? before + data.value + after : data.value;
```

 I also wanna make sure the generated code will have`<pre>` and `<code>`, not only a `<pre>`

Reply:
> I also wanna make sure the generated code will have`<pre>` and `<code>`, not only a `<pre>`

shouldn't `hljs: false` + `wrap: false` also generate `<pre><code>`? That was the purpose of https://github.com/hexojs/hexo-util/pull/112.

Reply:
Yes, after this PR [hexojs/hexo-util#112](https://github.com/hexojs/hexo-util/pull/112), the `hljs` is not needed anymore, but in the older version of that code, `hljs` is needed for `<code>`. Well, because there are two projects for `hexo` and `hexo-utils`, I think it is much better to make some compatibility codes in case the user mixed-use the new version hexo and old version hexo-utils.

Reply:
But enabling `hljs` wouldn't work with most theme which doesn't use the newer `hljs-*` syntax though.

I also created [a PR](https://github.com/hexojs/hexo-util/pull/138) to decouple `hljs:` and `wrap:`, i.e. enabling `hljs` will not disable `wrap`.

> in case the user mixed-use the new version hexo and old version hexo-utils.

we actually prefer plugins/themes to support older hexo (if the required workaround is minimal), than supporting older plugins.

Reply:
Updated.